WebPortal/Pages/Landing/LandingContent.razor

```diff
@@ -8,7 +8,7 @@
 @inject NavigationManager NavigationManager
 @inject IOptions<DataProjectsConfiguration> DataProjectsConfiguration
 
-<AeTypography Variant="h4" style="margin-top: 2.5rem">@Localizer["My Places"]</AeTypography>
+<MudText Typo="Typo.h6" Class="mt-12">My Places</MudText>
```

Comment:
This should stay localized. Same thing in RecentLinks.razor

Reply:
thank YOU